requirements-dev-test.txt

```diff
@@ -3,7 +3,6 @@
 # To run tests for smaller subsets of infrastructure, please look at other requirements-dev-*.txt files.
 # Otherwise (i.e., if/when you are not concerned with running tests), please ignore these comments.
 
-black==19.10b0  # lint
```

Comment:
👍 👍 👍  I came to the same conclusion in #2183 Apologies that I missed your PR before creating it.